torch/csrc/jit/frontend/sugared_value.h

```diff
@@ -284,8 +284,17 @@ struct TORCH_API BuiltinModule : public SugaredValue {
       // methods under its module.
       return std::make_shared<BuiltinModule>("aten", version);
     }
+
```

Comment:
All of these changes are to make `torch.ops...` resolve correctly, which isn't possible with the `NativeResolver`. To be honest I kinda guessed, so I wouldn't be surprised if I missed something here.